customers/views.py

```diff
@@ -93,16 +66,8 @@ def update(customer_id):
             updated_customer.last_name = request_data["last_name"]
             updated_customer.email = request_data["email"]
             updated_customer.dob = request_data["dob"]
-            db.session.commit()
-            return {
-                "id": f"{updated_customer.id}",
-                "username": f"{updated_customer.username}",
-                "first_name": f"{updated_customer.first_name}",
-                "last_name": f"{updated_customer.last_name}",
-                "email": f"{updated_customer.email}",
-                "dob": f"{updated_customer.dob}",
-                "date_created": f"{updated_customer.date_created}",
-            }, 200
+            updated_customer.save()
+            return (customerSchema.jsonify(updated_customer)), 200
         elif request.method == "PATCH":
             if "username" in request_data:
```

Comment:
again, how can you reduce this code by using the values from `request_date` more efficiently with your schema?